runtime/oti/j9.h

```diff
@@ -349,6 +349,10 @@ typedef struct {
 	char data[LITERAL_STRLEN(J9_UNMODIFIABLE_CLASS_ANNOTATION)];
 } J9_UNMODIFIABLE_CLASS_ANNOTATION_DATA;
 
+#if defined(J9VM_ZOS_3164_INTEROPERABILITY)
+#define J9_IS_31BIT_INTEROP_TARGET(handle) J9_ARE_ALL_BITS_SET((UDATA)handle, OMRPORT_SL_ZOS_31BIT_TARGET_HIGHTAG)
```

Comment:
handle needs to be bracketted.

Reply:
Good catch.  Updated in https://github.com/eclipse/openj9/pull/12074/commits/cfd0e84fed6fb508a2bc233bf2a057cffed13603 .  Thanks!